Sales-Engineering.md

```diff
@@ -0,0 +1,108 @@
+- I am discussing what I'm learning about or experimenting on with my teammates, maybe even collaborating on the experience.
+- I take my individual growth seriously and realize it is valuable to both myself and Octopus.
+- I reach out to my peers and leadership group to develop my growth plan.
+- I am a self-managing team member.
+- I make and keep my commitments using the privilege of flexibility wisely.
+
+### I am an egoless, team-oriented engineer
+
+- Other team members genuinely want me on their team.
+- I happily take responsibility for what I produce.
+- I avoid going dark and know when to call out for help.
+- I communicate professionally and keep stakeholders engaged at all times.
+- I am well aware of all processes of the team and diligently follow them.
+- Even when I disagree with something, I continue to work enthusiastically, learning from the others on the team.
+- Every venture has mundane tasks, and I work happily and diligently through them.
+- I go out of my way to develop a friendly and professional rapport with my customers and their team members.
+- My contributions to team discussions are net-positive, informative, and helpful.
+
+### I am learning how to help prospective and existing customers
+
+- I am learning about Octopus Deploy and its value proposition for our customers.
+- I am learning how to help customers discover if Octopus Deploy is the right tool for them.
+- I am learning how to help customers get the most out of their Octopus Deploy instance.
+- I pair with a senior member of the team to answer customer questions and provide guidance.
+- I advocate on behalf of my customers within my team.
+
+## Senior Solutions Engineer
+
+> I am a valuable and enthusiastic member of an Octopus team. I show great potential and have a high growth trajectory toward independent contribution.
+>
+> I have a broad knowledge of Continuous Delivery or direct experience with Octopus Deploy.
+
+### I help prospective and existing customers
+
+- I can explain Octopus Deploy and its value proposition to our customers.
+- I help customers discover if Octopus Deploy is the right tool for them.
+- I help customers get the most out of their Octopus Deploy instance.
+- I pair with a senior member of the team to answer customer questions.
+- I advocate on behalf of my customers to other Octopus teams.
+
+## Solutions Architect
+
+> I am a highly sought-after member of an Octopus team with a proven record of providing world-class guidance to our customers.
+>
+> I have extensive knowledge of Continuous Delivery or extensive experience with Octopus Deploy.
+
+### I teach my team to help prospective and existing customers
+
+- I train teammates on Octopus features and how they impact customers.
+- I work with teammates on how to best showcase features or solve customer needs.
+- I pair with other members of the team to help them answer customer questions.
+- I help team members advocate on behalf of our customers to other Octopus teams.
```

Comment:
I'm not sure of the right wording, but maybe also something like "I work with the customers to help them learn more about the best way to use Octopus and how different features can work for them."  And maybe "I work with the customer teams to assist in diversified use of Octopus" - for something about helping customers with getting Octopus usage up across different dev teams?  And different versions of those for the different levels of role.